### PR TITLE
hw 5.1

### PR DESCRIPTION
**Pull request by Djeckkenni**

### CODE REVIEW

File: src/main/java/core/basesyntax/Car.java [hunk 1]
```diff
@@ -1,5 +1,77 @@
...
+        return airBag;
+    }
+
+    public Engine getEngine() {
+        return engine;
+    }
+
+    @Override
+    public Car clone() throws CloneNotSupportedException {
+        Car cloned = (Car) super.clone();
```
Comment (Tarabarnik) [inline src/main/java/core/basesyntax/Car.java:73]: please see the checklist to understand what's wrong here

File: src/main/java/core/basesyntax/Engine.java [hunk 2]
```diff
@@ -1,5 +1,62 @@
...
+        return accelerationTime;
+    }
+
+    public int getTorque() {
+        return torque;
+    }
+
+    @Override
+    public Engine clone() throws CloneNotSupportedException {
+        return (Engine) super.clone();
```
Comment (Tarabarnik) [inline src/main/java/core/basesyntax/Engine.java:60]: same

File: src/main/java/core/basesyntax/Car.java (hunk 1 above)
Comment (Djeckkenni) [reply src/main/java/core/basesyntax/Car.java]: Done

File: src/main/java/core/basesyntax/Engine.java (hunk 2 above)
Comment (Djeckkenni) [reply src/main/java/core/basesyntax/Engine.java]: Done

File: src/main/java/core/basesyntax/Car.java [hunk 3]
```diff
@@ -1,5 +1,29 @@
...
+        this.yearOfIssue = yearOfIssue;
+        this.airBag = airBag;
+        this.engine = engine;
+    }
+
+    @Override
+    public Car clone() {
+        Car cloned = clone();
+        cloned.engine.clone();
+        return cloned;
```
Comment (Tarabarnik) [inline src/main/java/core/basesyntax/Car.java:27]: try to call this clone from main

File: src/main/java/core/basesyntax/Engine.java [hunk 4]
```diff
@@ -1,5 +1,22 @@
...
+        this.name = name;
+        this.volume = volume;
+        this.power = power;
+        this.accelerationTime = accelerationTime;
+        this.torque = torque;
+    }
+
+    @Override
+    public Engine clone() {
+        return clone();
```
Comment (Tarabarnik) [inline src/main/java/core/basesyntax/Engine.java:20]: same

File: src/main/java/core/basesyntax/Car.java (hunk 3 above)
Comment (Djeckkenni) [reply src/main/java/core/basesyntax/Car.java]: Done

File: src/main/java/core/basesyntax/Engine.java (hunk 4 above)
Comment (Djeckkenni) [reply src/main/java/core/basesyntax/Engine.java]: Done

**Pull request closed without merging**